pmml/pmml.py

```diff
@@ -214,5 +210,5 @@ def print_pmml(pmml_out):
             print("\t\t</Segmentation>", file=pmml_out)
             print("\t</MiningModel>", file=pmml_out)
             print("</PMML>", file=pmml_out)
-except Exception as ioex:
+except IOError as ioex:
```

Comment:
and you can close model_in after you read it. so remove one indent to left from `model_content = filter(...)` line.